plugins/no_sir.py

```diff
@@ -0,0 +1,20 @@
+import re
+from errbot import BotPlugin
+
+
+class no_sir(BotPlugin):
```

Comment:
class name not starting with caps :scream:

Reply:
Thats by mistake @meetmangukiya i will correct all the issues now in this pr